doc/guides/using-symbols.md

```diff
@@ -0,0 +1,61 @@
+# Using global symbols
+
+ES6 introduced a new type: `Symbol`. This new type is _immutable_, and
+it is often used for metaprogramming purposes, as they can be used as
+property keys like string. There are two types of
+symbols, local and globals. Symbols are not included in
+JSON.stringify() but thay are by `util.inspect()`.
+
+## Local `Symbol`
+
+Local symbols are created with `Symbol(string)` and they are local to
+the caller function. Note that `Symbol('hello') !== Symbol('hello')`.
+For this reason, we often use them to simulate private fields, like so:
+
+```js
+const kField = Symbol('field');
```

Comment:
Please make this `Symbol('kField')`. Having consistent naming is *so* helpful for `grep`-ping, and I’ve been changing it to this pattern whenever I touched these lines.